docs/can-guides/commitment/recipes/canvas-clock/canvas-clock.md

```diff
@@ -14,9 +14,140 @@ In this guide you will learn how to:
 
 The final widget looks like:
 
-<a class="jsbin-embed" href="https://jsbin.com/jeligek/3/embed?output">
-  CanJS Canvas Clock on jsbin.com
-</a>
+```html
+<style>
```

Comment:
I think we should still use CodePen to embed the starter and finished code examples, like we do in this guide: https://canjs.com/doc/guides/recipes/signup-simple.html